packages/flutter/lib/src/cupertino/route.dart

```diff
@@ -389,97 +395,96 @@ class CupertinoPageTransition extends StatefulWidget {
     required this.linearTransition,
   });
 
-  /// The widget below this widget in the tree.
-  final Widget child;
-
-  ///  * `primaryRouteAnimation` is a linear route animation from 0.0 to 1.0
-  ///    when this screen is being pushed.
+  /// A linear route animation from 0.0 to 1.0 when this screen is being pushed.
   final Animation<double> primaryRouteAnimation;
 
-  ///  * `secondaryRouteAnimation` is a linear route animation from 0.0 to 1.0
-  ///    when another screen is being pushed on top of this one.
+  /// A linear route animation from 0.0 to 1.0 when another screen is being
+  /// pushed on top of this one.
   final Animation<double> secondaryRouteAnimation;
 
-  ///  * `linearTransition` is whether to perform the transitions linearly.
-  ///    Used to precisely track back gesture drags.
+  /// Whether to perform the transitions linearly. Used to precisely track back
+  /// gesture drags.
   final bool linearTransition;
 
+
+  /// The widget below this widget in the tree.
+  final Widget child;
+
   @override
   State<CupertinoPageTransition> createState() => _CupertinoPageTransitionState();
 }
 
 class _CupertinoPageTransitionState extends State<CupertinoPageTransition> {
 
+  CurvedAnimation? _primaryPositionCurvedAnimation;
 
-  // When this page is coming in to cover another page.
+  /// When this page is coming in to cover another page.
   late Animation<Offset> _primaryPositionAnimation;
-  // When this page is becoming covered by another page.
+
+  CurvedAnimation? _secondaryPositionCurvedAnimation;
+
+  /// When this page is becoming covered by another page.
```

Comment:
Since these are private, the `///` public docs do not need to be added. For the others, can you add the comments back?